ws-service/server.ts

```diff
@@ -44,25 +44,55 @@ const addressRouteConnection = (socket: Socket): void => {
   console.log('  total:', countA)
 }
 
+const resolveOpReturn = (opr: any) => {
+  try {
+    if (!(typeof opr === 'string' && opr !== '')) {
+      throw new Error("Error parsing Opreturn options")
+    }
+    return JSON.parse(opr)
+  } catch (e) {
+    return undefined
+  }
+}
+
 const broadcastTxs = async (broadcastTxData: BroadcastTxData): Promise<void> => {
   console.log('broadcasting', broadcastTxData.txs.length, broadcastTxData.messageType, 'txs to', broadcastTxData.address)
   try {
-    const parsedTxs = broadcastTxData.txs.map(
+    const { address, messageType, txs } = broadcastTxData
+    const broadcastedTransactions: BroadcastTxData = {
```

Comment:
not a better variable name: the broadcasted transactions are a specific field of this data object (namely: `.txs`)

Reply:
so broadcastedTxs is a better name ? sure